tests/src/test/java/org/eclipse/hono/tests/IntegrationTestSupport.java

```diff
@@ -343,6 +341,23 @@ public String getRandomDeviceId(final String tenantId) {
         });
     }
 
+    /**
+     * Returns a command client for a device.
+     *
+     * @param tenantId The tenant that the device belongs to.
+     * @param deviceId The identifier of the device.
+     * @param requestTimeout The number of milliseconds to wait for a response from the device.
+     * @return A future that may contain a command client.
+     */
+    public Future<CommandClient> getCommandClient(final String tenantId, final String deviceId,
```

Comment:
this change also seems to be unrelated to this PR, or isn't it? If it is, please remove it.